dashboard/src/main/resources/templates/dashboard.ftl

```diff
@@ -16,19 +16,23 @@
           <span class="desc">Total Artifacts Checked</span>
         </div>
         <div class="statistic-item statistic-item-red">
-          <h2 class="artifact-count"><@countFailures name="Static Linkage Errors"/></h2>
+          <h2 class="artifact-count">${dashboardMain.countFailures(table, "Static Linkage Errors")}
```

Comment:
Could you (not necessarily in this PR) use selector `statistic-item > h2` instead of a separate `artifact-count` class?